temba/campaigns/views.py

```diff
@@ -431,7 +431,7 @@ def __init__(self, user, *args, **kwargs):
                 # otherwise, its just a normal language
                 initial = message.get(language.iso_code, "")
 
-            field = forms.CharField(widget=forms.Textarea, required=False, label=language.name, initial=initial)
+            field = forms.CharField(widget=CompletionTextarea(attrs={"widget_only": True}), required=False, label=language.name, initial=initial)
```

Comment:
@ihor-palii - if I'm not mistaken, I changed this field some while ago to make it work with the new version of the Flow Parameters feature (https://github.com/greatnonprofits-nfp/rapidpro/commit/ceb1f7b9e22a784674eb96c3f909032c1fd80ba9#diff-5a29020a25af44cdd1d3ccda58a0adc599bb980ebe016bd848d725752a460cccL472). So before we merge it, can you please check if the Flow Parameters feature is working with `CompletionTextarea` widget? Thanks.

Reply:
@teehamaral  I rechecked the behavior of flow params and it works as previously (attached video to the ticket). I used `CompletionTextarea` only for the `languages` field which is used only inside of `#message-section`. Also from the code, we can see that we don't use flow params when `#message-section` activated https://github.com/greatnonprofits-nfp/rapidpro/commit/ceb1f7b9e22a784674eb96c3f909032c1fd80ba9#diff-fb385162f0ece20f6adf4f16044de2fdf6985227f4b41cac555185c274666526R62.